src/createHandler.ts

```diff
@@ -42,10 +48,12 @@ export default function createHandler(parser: SaxesParser, options: SaxesOptions
 	// Helpers for other responsibilities
 	const namespaces = createNamespaceContext(options.additionalNamespaces || {});
 
-	const [track, trackClose, update] = options.position
+	const [track, trackClose, update, getNextPosition, trackAttribute] = options.position
 		? createPositionTracker(parser)
 		: positionTrackerStubs;
 
+	let attributePositions: Map<string, NextPosition> = new Map()
```

Comment:
I do wonder what will happen if there are two attributes with the same name + namespace (I think usually the first of the duplicates is used as the "real" value?). 

I guess it depends on the order in which `Object.keys(element.attributes)` iterates through the attributes, but maybe only the position of the first occurrence should be stored?

Reply:
`saxes` would throw an error; `duplicate attribute: …`

I've added an assertion to check this and to keep it in check if the `saxes` dependency is ever updated/replaced.